<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <widget class="QWidget" name="Login_page">
     <widget class="QPushButton" name="SignUp_pb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AdminLogIn_pb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserLogIn_pb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SignUp_page">
     <widget class="QLineEdit" name="SignUpName_Input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SignUpMail_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SignUp_Mobile">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mobile Number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SignUp_pass">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SignUp_pb_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LoginAsAdmin_page">
     <widget class="QLineEdit" name="LogIn_AdminMail_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LogIn_AdminPass_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Enter password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AdminLogin_pb">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>500</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LoginAsUser_Page">
     <widget class="QLineEdit" name="Login_UserMail_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Login_UserPass_input">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserLogin_pb">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>log in</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="UserHomePage">
     <widget class="QPushButton" name="Booking_pb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Book ticket and Add Ride</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit1_pb">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>440</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Nightstay_pb">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Night stay</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="BookTicketPage">
     <widget class="QLineEdit" name="Booking_DayInput">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Day</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Booking_MonthInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Month</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Booking_YearInput">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Year</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Booking_QuantityInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Quantity</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back1_pb">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continue_pb">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>510</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>continue</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="BumperCar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>330</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Bumper Car</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Train_Ride">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>370</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Train Ride</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Roller_Coaster">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Roller coaster</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter date &amp; ticket quantity*</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>270</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add ride quantity as you want</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="continuePage">
     <widget class="QPushButton" name="confirm_pb">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Total payable ammont is 
</string>
      </property>
     </widget>
     <widget class="QLabel" name="continue_total_price">
      <property name="geometry">
       <rect>
        <x>402</x>
        <y>189</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminHomePage">
     <widget class="QPushButton" name="A_ShowAllUser_pb">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Show all user</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_remove_pb">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_UpdatePrice_pb">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Update price</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Histroy_pb">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Booking History</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Search User</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ShowAllUserPage">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>761</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SearchUserPage">
     <widget class="QPushButton" name="A_Enter_pb">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="A_searchMail_input">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter mail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Home_pb">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RemoveUserPage">
     <widget class="QLineEdit" name="A_removeMail_input">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter mail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Enter_pb_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Home_pb_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>490</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="UpdatePricePage">
     <widget class="QLineEdit" name="A_UpdatePrice_input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>251</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Updated Ticket Price</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Enter_pb_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Home_pb_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>490</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="A_UpdateBumper">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>251</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Update BumperCar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="A_UpdateTrain">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>251</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Update TrainRide</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="A_UpdateRollerCoaster">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Update RollerCoaster</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="BookingHistoryPage">
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>771</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StayPage">
     <widget class="QLineEdit" name="people">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>People</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stay">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Days</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmpb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>370</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
